caravel/static/js/continuation.js

```diff
@@ -50,8 +57,17 @@ var fetchMore = coalesce(function(complete) {
             return;
 
         // Append the results to the existing results <div>
-        var frag = document.createElement("div");
+        if (viewQuery == "v=List") {
+            var frag = document.createElement("table");
+            var att = document.createAttribute("class");
+            att.value = "table listings-table"
+            frag.setAttributeNode(att)
```

Comment:
1. I think it should be called frag, because it isn't a table in the thumbnail view. Like this is a generic continuation fragment. 
2. Fixed.